### ISSUE TITLE
Narrator connot copy and paste in EPUB reading systems.

### ISSUE THREAD

**Issue by GeorgeKerscher** [daisy/reading-system-testing#10]
https://github.com/user-attachments/assets/162a29a2-84dc-4ce1-a8eb-e9b879d3c877

In our reading system testing, one test is to copy and paste. We have found that when using Narrator this test fails on several reading systems. The same reading systems pass this test with Jaws and NVDA.

We have tested this  on a Microsoft Surface running Windows Version 25H2 (OS Build 26220.7535)  This is running an insider build.
 Edge:144.0.3719.67 (Official build) beta (arm64) 

We get the same behavior on a Lenovo P1  running 25H2 (OS Build 26200.7462) 
Edge: 143.0.3650.139 (Official build) (64-bit) 

We get the same behavior when using Chrome. The Applications that show this behavior are:
[Bookshare Web reader](https://www.bookshare.org)

[VitalSource Web reader](https://www.vitalsource.com/)
[and Thorium Windows application](https://github.com/edrlab/thorium-reader/releases/tag/v3.3.0)

I believe these three applications use a WebView for rendering the content.

**Comment by prashantverma2014** [comment]
The text copy issue with Narrator is confirmed on HP laptop running Windows 11 Pro Version 10.0.26200 Build 26200 and Microsoft Edge Version 144.0.3719.82 (Official build) (64-bit)

The text copy and paste is not working in Thorium Reader app version 3.3.0 and Bookshare Web Reader with Narrator. 
In this same environment text copy and paste is working with Jaws.

**Comment by GeorgeKerscher** [comment]
I submitted this through EDAD. I was given a short link to the submission:

https://aka.ms/AAu2xjj
